mlrun/feature_store/retrieval/storey_merger.py

```diff
@@ -146,27 +145,26 @@ def init_online_vector_service(
         server = create_graph_server(graph=graph, parameters={})
 
         cache = ResourceCache()
+        all_fs_entities = []
         for featureset in feature_set_objects.values():
             driver = get_online_target(featureset)
             if not driver:
                 raise mlrun.errors.MLRunInvalidArgumentError(
                     f"resource {featureset.uri} does not have an online data target"
                 )
             cache.cache_table(featureset.uri, driver.get_table_object())
+
             for key in featureset.spec.entities.keys():
-                if (
-                    not self.vector.spec.with_indexes
```

Comment:
why did you remove this line?

Reply:
It is not really necessary there because inside the `get` function in line 529 I check it before delete those columns from the response.